### ISSUE TITLE
Blurry Background Available in Main

### ISSUE THREAD

**Issue by Congyuwang** [skarline/zed-xcode-themes#2]
This is a wonderful theme, and now easily my favorite theme. Thank you a lot for authoring it!

Now, zed has this nice PR https://github.com/zed-industries/zed/pull/9610 that enables blurry / transparent background.

I'm thinking that perhaps following Xcode style, we can have certain parts of the editor to assume dark/light blurriness. Although I have yet only limited idea of the specifics.

**Comment by skarline** [comment]
Thank you so much for your kind words! I'm thrilled to hear you're enjoying it!

I've been really looking forward for that PR to be merged! As of now, I'm exploring how to integrate this feature in a way that is cohesive with the rest of the Zed UI. Hopefully, more themes will start implementing it soon enough.

If you, or anyone else, have any ideas of what it should look like, feel free to provide a `theme_overrides` configuration for one of the themes so we can discuss on it!

**Comment by Congyuwang** [comment]
I've tried to experiment with theme_overrides and got the following settings that seems to give me a Xcode feeling:
- for Xcode Default Light
```json
  // for Xcode Default Light
  "experimental.theme_overrides": {
    "background.appearance": "blurred",
    "background": "#e8e8e8c0",
    "border": "#909090c0",
    "text": "#000000ff",
    "text.muted": "#404040ff",
    // problem: what is text.disabled / ghost_element.disabled
    // "text.disabled": "#808080ff",
    "status_bar.background": "#00000000",
    "title_bar.background": "#00000000",
    "panel.background": "#00000000",
    "ghost_element.background": "#00000000",
    "ghost_element.hover": "#00000020",
    "ghost_element.active": "#00000020",
    "ghost_element.selected": "#00000020"
  }
```
<img width="1470" alt="截圖 2024-04-10 20 39 02" src="https://github.com/skarline/zed-xcode-themes/assets/52687642/1539be44-0d9a-42ee-880d-cd4a2dd2192e">

<img width="1470" alt="截圖 2024-04-10 20 40 22" src="https://github.com/skarline/zed-xcode-themes/assets/52687642/94e6faab-c21c-4242-927d-fb85557efcaa">

- for Xcode Default Dark
```json
  // for Xcode Default Dark
  "experimental.theme_overrides": {
    "background.appearance": "blurred",
    "background": "#2a2a30c0",
    "border": "#101010ff",
    "text": "#ffffffff",
    "text.muted": "#bfbfbfff",
    // problem: what is text.disabled / ghost_element.disabled
    // "text.disabled": "#something",
    "status_bar.background": "#00000000",
    "title_bar.background": "#00000000",
    "panel.background": "#00000000",
    "ghost_element.background": "#00000000",
    "ghost_element.hover": "#a8a8a880",
    "ghost_element.active": "#a8a8a880",
    "ghost_element.selected": "#a8a8a880"
  }
```
<img width="1470" alt="截圖 2024-04-10 20 48 57" src="https://github.com/skarline/zed-xcode-themes/assets/52687642/816c2c44-5be5-4d62-b5c0-3559fbfc1e11">

<img width="1470" alt="截圖 2024-04-10 20 48 42" src="https://github.com/skarline/zed-xcode-themes/assets/52687642/7e00eba5-1d4d-4436-b476-7ee4b91ee784">

Mainly, I've changed the alpha value of panel / status_bar / title_bar backgrounds.
Also I adjusted some text colors to make the text more obvious.
I've also adjusted the ghost element color to make it more like Xcode.
It look good to me. But I am not fully familiar with Zed settings, nor confident in UI color design at all! 😀

Give it a try! And see if there is anything that needs further refinement.

**Comment by Congyuwang** [comment]
I don't know how to adjust the background of selected items in the project panel. For the moment they are opaque against the blurry panel background. I think I'd rather them to have some alpha, to make it look more harmonious with its background.

**Comment by jmshgs** [comment]
Just wanna say these look amazing!

**Comment by SergoGansta777** [comment]
And so, it turns out, there are no official blurry background versions planned?

**Comment by skarline** [comment]
@SergoGansta777  Not really. I have experimented with it in the past; however, I was never satisfied with the result. Also, a couple of Zed updates broke much of the transparency stuff, so I decided to put it off until it becomes more stable :/

Anyway, I'm open to experimenting with it sometime. So, feel free to suggest any transparent themes that you like.
